### PR TITLE
Update changelog

### PR DESCRIPTION
**Pull request by GomathiselviS**
Signed-off-by: GomathiselviS <gomathiselvi@gmail.com>

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -108,3 +108,31 @@ releases:
         name: eos
         namespace: null
     release_date: '2020-06-23'
+  1.0.2:
```
Comment (NilashishC) [inline changelogs/changelog.yaml:111]:
```suggestion
  1.0.1:
```

File: changelogs/changelog.yaml [hunk 2]
```diff
@@ -108,3 +108,31 @@ releases:
...
+      - Make `src`, `backup` and `backup_options` in eos_config work when module alias
+        is used (https://github.com/ansible-collections/arista.eos/pull/85).
+      minor_changes:
+      - Add changelogs for 1.0.0 release (https://github.com/ansible-collections/arista.eos/pull/62)
+      - Add round trip testcases to the 2.9 resource modules.
+      - Add unit testcases to the eos_l3_interfaces resource modules.
+      - Add unit testcases to the eos_lag_interfaces resource modules.
+      - Sorted the list of params of ip address before forming the tuple.
+      - Updated docs.
+      release_summary:
```
Comment (NilashishC) [inline changelogs/changelog.yaml:126]: I believe the all the changelogs would go under 1.0.1 and the 1.0.2 would just have a release_summary entry that mentions that it's a re-release of 1.0.1 with updated changelog.

File: changelogs/CHANGELOG.rst [hunk 3]
```diff
@@ -5,6 +5,33 @@ Arista Eos Collection Release Notes
 .. contents:: Topics
 
 
+v1.0.2
+======
+
+Release Summary
+---------------
+
+- rereleasing 1.0.2 with updated changelog.
```
Comment (rohitthakur2590) [inline changelogs/CHANGELOG.rst:14]: aren't this supposed to be rereleasing 1.0.1?

File: changelogs/changelog.yaml [hunk 4]
```diff
@@ -108,3 +108,35 @@ releases:
...
+    - 80-linkagg-name-parameter.yaml
+    - 89-add-ut-l3_interfaces.yaml
+    - 90-add-ut-lag-interfaces.yaml
+    - 93-update-docs.yaml
+    - fix_src_backup_with_module_alias.yaml
+    release_date: '2020-07-31'
+  1.0.2:
+    changes:
+      release_summary:
+      - rereleasing 1.0.2 with updated changelog.
```
Comment (rohitthakur2590) [inline changelogs/changelog.yaml:139]: same as above

**Pull request merged**